src/main/java/org/tdl/vireo/service/SystemDataLoader.java

```diff
@@ -524,82 +516,63 @@ private List<WorkflowStep> processWorkflowSteps(Organization organization, List<
             newWorkflowStep.setInstructions(workflowStep.getInstructions());
 
             newWorkflowStep = workflowStepRepo.save(newWorkflowStep);
-
-            workflowSteps.add(newWorkflowStep);
         }
-        return workflowSteps;
     }
 
-    private void processEmailWorflowRules() {
-
-        Organization organization = organizationRepo.findById(1L).get();
+    private void processEmailWorflowRules(Organization organization, Organization systemOrganization) {
+        List<EmailWorkflowRule> emailWorkflowRules = organization.getEmailWorkflowRules();
```

Comment:
Gets the already persisted email workflow rules from the initial institute (top-level) organization.